rest_framework_simplejwt/backends.py

```diff
@@ -39,6 +50,15 @@ def _validate_algorithm(self, algorithm):
         if algorithm in algorithms.requires_cryptography and not algorithms.has_crypto:
             raise TokenBackendError(format_lazy(_("You must have cryptography installed to use {}."), algorithm))
 
+    def get_verifying_key(self, token):
+        if self.algorithm.startswith("HS"):
+            return self.signing_key
+
+        if self.jwks_client:
+            return self.jwks_client.get_signing_key_from_jwt(token).key
```

Comment:
@Andrew-Chen-Wang Heya, I had a question about this part if you don't mind me asking but is there a reason we are getting the signing key from the jwt as the verifying key? Shouldn't it get the public key from the jwk-uri instead of the private/signing key? (  I had a jwk endpoint that I was using that only returns the public key to be used for verification but it doesn't work since this expects the signing key )

Reply:
We're trying to decode the token; naming is a little weird ngl https://pyjwt.readthedocs.io/en/stable/usage.html#retrieve-rsa-signing-keys-from-a-jwks-endpoint

Reply:
Ahh I see, thanks and sorry for the trouble!

Reply:
np and no worries!